src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -214,23 +216,47 @@ class JesBackend extends Backend with LazyLogging {
     }
   }
 
-  private def uploadCommandScript(backendCall: BackendCall, command: String): Unit = {
+  private def uploadCommandScript(backendCall: BackendCall, command: String): Try[Unit] = {
     val fileContent =
       s"""
          |#!/bin/bash
          |$command
          |echo $$? > ${JesBackendCall.RcFilename}
        """.stripMargin.trim
-    JesConnection.storage.uploadObject(backendCall.gcsExecPath, fileContent)
+
+    def attemptToUploadObject(priorAttempt: Option[Unit]) = JesConnection.storage.uploadObject(backendCall.gcsExecPath, fileContent)
+
+    TryUtil.retryBlock(
+      fn = attemptToUploadObject,
+      retries = 10,
+      pollingInterval = 5 seconds,
+      pollingBackOffFactor = 1,
+      maxPollingInterval = 10 seconds,
+      failMessage = Some(s"${makeTag(backendCall)} Exception occurred while uploading script to ${backendCall.gcsExecPath}")
+    )
   }
 
-  private def runWithJes(backendCall: BackendCall, command: String, jesInputs: Seq[JesInput], jesOutputs: Seq[JesOutput]): ExecutionResult = {
-    val tag = makeTag(backendCall)
-    val jesParameters = backendCall.standardParameters ++ gcsAuthParameter(backendCall.workflowDescriptor) ++ jesInputs ++ jesOutputs
-    logger.info(s"$tag `$command`")
-    uploadCommandScript(backendCall, command)
+  private def createJesRun(backendCall: BackendCall, jesParameters: Seq[JesParameter]): Try[Run] = {
+    def attemptToCreateJesRun(priorAttempt: Option[Run]): Run = Pipeline(
+      backendCall.jesCommandLine,
+      backendCall.workflowDescriptor,
+      backendCall.key,
+      jesParameters,
+      googleProject(backendCall.workflowDescriptor),
+      JesConnection
+    ).run
+
+    TryUtil.retryBlock(
+      fn = attemptToCreateJesRun,
+      retries = 10,
+      pollingInterval = 5 seconds,
+      pollingBackOffFactor = 1,
+      maxPollingInterval = 10 seconds,
+      failMessage = Some(s"${makeTag(backendCall)} Exception occurred while creating JES Run")
+    )
```

Comment:
hmm I suppose...  are the values for polling interval, back-off factor, etc going to be the same for the foreseeable future?